providers/amazon/tests/system/amazon/aws/example_mongo_to_s3.py

```diff
@@ -16,14 +16,13 @@
 # under the License.
 from __future__ import annotations
 
-from providers.amazon.tests.system.amazon.aws.utils import SystemTestContextBuilder
-
 from airflow.models.baseoperator import chain
 from airflow.models.dag import DAG
 from airflow.providers.amazon.aws.operators.s3 import S3CreateBucketOperator, S3DeleteBucketOperator
 from airflow.providers.amazon.aws.transfers.mongo_to_s3 import MongoToS3Operator
 from airflow.utils.timezone import datetime
 from airflow.utils.trigger_rule import TriggerRule
+from system.amazon.aws.utils import SystemTestContextBuilder
```

Comment:
Most of similar changes are result of applyin PEP-8 import order on imports afre we cnanged the "from"